test/web-auth/popup.test.js

```diff
@@ -486,6 +478,25 @@ describe('auth0.WebAuth.popup', function() {
         });
       });
 
+      var expectedOptions = {
+        email: 'me@example.com',
+        password: '123456',
+        scope: 'openid',
+        realm: 'the_connection',
+        popup: true,
+        responseType: ''
+      };
+      stub(TransactionManager.prototype, 'process', function(options) {
+        delete options.popupHandler;
```

Comment:
Feels like cheating. Rather than removing it I would check that it's actually present.

Reply:
we can't compare in the same way because it has functions and the `expect(...).to.be.eql(..)` doesn't work with functions. In any case, we remove popupHandler in the next method.

Reply:
then first check if `options.popupHandler` exists and is !== null/undefined, then remove it, and then do the optA.to.be.eql(optB)

Reply:
what for? I don't care about the popupHandler at all, that's why I'm not asserting anything related to it.

Reply:
I don't know. I thought that checking that is another way of ensuring that you're using the popup mode. But if you think it's worthless then don't. 👍